__tests__/nc-games.test.js

```diff
@@ -90,4 +90,20 @@ describe('GET /api/users', () => {
                 })
             });
     })
+})
+
+describe('PATCH /api/reviews/:review_id', () => {
+    test('returns the updated review which has incremented vote', () => {
+        const review_id = 2;
+        const incVote = { inc_votes: 10 }
+
+        return request(app)
+            .patch(`/api/reviews/${review_id}`)
+            .send(incVote)
+            .expect(200)
+            .then((res) => {
+                expect(res.body.review).toHaveProperty("votes", 15);
+                expect(res.body.review).toHaveProperty("review_id", 2)
+            })
+        })
```

Comment:
In addition to this test, we would want..:
- a test for when the review isn't found
- a test for when the review id is invalid
- a test for when the passed inc_votes is not an integer
- and a test for when the inc_vote key is missing! For this one, you can either respond with a 200 (and make no change to the review) or return a 400 (and return an error)